### ISSUE TITLE
[Windows] Not a terminal error when shift key is pressed

### ISSUE THREAD

**Issue by spenserblack** [mitsuhiko/console#83]
When reading a line from the terminal, pressing the shift key results in an error `{ kind: NotConnected, error: "Not a terminal" }` on Windows. It seems to happen with both Powershell and Command Prompt

# To Reproduce

(In Windows)

```bash
git clone https://github.com/mitsuhiko/console.git
cargo run --example term
# press shift key on the To edit prompt
```

**Comment by mitsuhiko** [comment]
This is caused by the very buggy `windows-console-colors` feature. I would recommend turning it off.

**Comment by spenserblack** [comment]
Thanks! Are there any plans to expose this feature in [dialoguer](https://github.com/mitsuhiko/dialoguer)? This seems to affect the `Input` prompt.

**Comment by mitsuhiko** [comment]
I will try to figure out what I can do here.

**Comment by pksunkara** [comment]
We will definitely fix these issues. I am just waiting to catch up with @mitsuhiko before proceeding.

**Comment by pksunkara** [comment]
This is fixed in 0.14.0 of console and will be propagated to dialoguer
